ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -146,6 +147,27 @@ func ResourceIBMPIInstance() *schema.Resource {
 				Default:     true,
 				Description: "Indicates if all volumes attached to the server must reside in the same storage pool",
 			},
+			Arg_DeploymentTarget: {
+				Description: "The deployment of a dedicated host",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						Attr_ID: {
+							Description: "The uuid of the host group or host",
+							Required:    true,
+							Type:        schema.TypeString,
+						},
+						Attr_Type: {
+							Description:  "specify if deploying to a host group or a host",
```

Comment:
Have description here match the description in the markdown?
```suggestion
							Description:  "The deployment target type. Supported values are 'host' and 'hostGroup'.",
```